src/proxy/mod.rs

```diff
@@ -23,7 +27,20 @@ impl Proxy {
 
 #[async_trait::async_trait]
 impl Provider for Proxy {
-    async fn create_invoice(&self, inv: &crate::Invoice) -> Result<Vec<crate::Label>> {
+    async fn create_invoice(
+        &self,
+        inv: &mut crate::Invoice,
+        _role: SignatureRole,
+        secret_key: &SecretKeyEntry,
+        strategy: VerificationStrategy,
+    ) -> Result<Vec<crate::Label>> {
+        // TODO: Need a way to load the local public keyring
```

Comment:
Maybe change the `new` function? Does this need to be addressed before merge?

Reply:
That was originally for #141... but I might just remove it now and re-do it later. Right now it basically does nothing.

Reply:
Actually... I am going to leave this for pairing. I am not sure I completely understand the Proxy setup

Reply:
Alright, I figured out what I was doing. It is part of #141 that I need to redo this. So I have removed it for now and left a TODO to do it along with #141.